homeassistant/components/motion_blinds/config_flow.py

```diff
@@ -17,10 +29,46 @@
 CONFIG_SETTINGS = vol.Schema(
     {
         vol.Required(CONF_API_KEY): vol.All(str, vol.Length(min=16, max=16)),
+        vol.Optional(CONF_INTERFACE, default=DEFAULT_INTERFACE): str,
     }
 )
 
 
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Options for the component."""
+
+    def __init__(self, config_entry: config_entries.ConfigEntry):
+        """Init object."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input=None):
+        """Manage the options."""
+        errors = {}
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        settings_schema = vol.Schema(
+            {
+                vol.Optional(
+                    CONF_WAIT_FOR_PUSH,
```

Comment:
Please explain why we need these options.

Reply:
@MartinHjelmare the current integration relays on multicast pushes to update it's data, however I have received multiple issues from users that have trouble with the multicast pushes because their network is blocking them (there router is blocking multicast messages, or the HomeAssistant device and the Motion Gateway are in diffrent VLAN's, or the multicast messages do not arrive with a cause that I/they could not find/solve).

However those multicast pushes are not strictly nessesarry since the gateway will also return the state of the blinds from its cache memory using direct communication. But that would mean we would need a higher update intervall giving unnesasarry RF communication between the gateway and the battery powered blinds, maybe draining there battery faster.

So the possibilites are:
1) default: CONF_WAIT_FOR_PUSH: True, CONF_FAST_UPDATE: False
Updates every 15 minutes where correct communication with the blind over RF is checked with the arrival of the multicast push, if the multicast push does not arrive after 5 tries of 5 seconds the entity is marked as unavailble (the battery might be dead, or the blind lost its communication with the gateway)

2) CONF_WAIT_FOR_PUSH: False, CONF_FAST_UPDATE: False
Updates every 15 minutes but the RF communication is not checked (multicast push), this means entities will never be marked as unavailble and you can opperate as normal, but the state is not garenteed to be up to date (can be lacking up to 30 minutes if multicast messages are not comming trough because the cache update always lacks behind one update command). This can be usefull if you do not want the entities beeing marked as unavailble whenever the RF communication fails (sometimes happens), normally in this case the gateway can still send commands to the blind and control it but the RF communication from the blind to the gateway fails, so the update command fails. If the entity is not marked as unavailble, it can still be controlled (although of course not garanteed if RF communication is bad).

3) CONF_WAIT_FOR_PUSH: False, CONF_FAST_UPDATE: True
Updates every 1 minute, this makes sure you have a fairly resent state even if multicast messages are beeing blocked.

Reply:
Ok, thanks.

Reply:
Any more suggestions, or is this ready for merge?